website/docs/discover-podman-desktop.md

```diff
@@ -0,0 +1,137 @@
+- Search, start, restart, stop, or delete a container.
+- Bulk deletion of selected containers.
+- Export a container to your local machine.
+- View the summary and logs of a container.
+- View the kube configuration of the container if it is a Kubernetes resource.
+- Interact with the container using a terminal.
+- View containers based on their status, whether they are running or stopped.
+- Remove all unused containers from the Podman engine for efficient memory usage.
+
+For more details, refer to [Working with containers](/docs/containers).
+
+#### Podify your containers
+
+![pods](img/pods-component.png)
+
+Access the **Pods** component in the UI to do the pod-related tasks:
+
+- Create a pod from a Kubernetes YAML file.
+- Search, start, restart, stop, or delete a pod.
+- View the status of containers that are part of the pod, whether they are running, created, exited, or in waiting state.
+- View the summary and logs of the pods
+- Generate and view configuration for a Kubernetes pod.
+- [Deploy a pod to a Kubernetes cluster](/docs/kubernetes/deploying-a-pod-to-kubernetes).
+- Interact with the pod using a terminal.
+- View pods based on their status, whether they are running or stopped.
+- Remove all unused pods from the Podman engine for efficient memory usage.
+
+#### Manage images
+
+![images](img/images-component.png)
+
+Access the **Images** component in the UI to do the image-related tasks:
+
+- Build an image from a container or docker file.
+- Pull an image from a registry.
+- Search or delete an image.
+- Create a container from the image.
+- Push an image to a configured registry.
+- Edit the image name and tag.
+- View the image history.
+- Save an image to your local machine.
+- Push an image to a cluster.
+- Import images from your local machine into the Podman Desktop application.
+
+#### Manage Kubernetes-based environment
+
+![kubernetes](img/kubernetes-component.png)
+
+Access the **Kubernetes** component in the UI to do the Kubernetes-related tasks:
+
+- Create an object after applying a kubernetes YAML configuration.
+- Search or delete an object.
+- View the summary and Kubernetes configuration of the object
+- Edit the configuration of the Kube object and apply those changes to your cluster directly from the UI.
+
+For more details, refer to [From containers to Kubernetes](/docs/kubernetes).
+
+#### Manage Docker compatibility
+
+Docker compatibility is a way to configure an environment in which you can run your Docker applications, commands, and tools on a Podman engine without reconfiguration. It encompasses two stages. You first [import your saved containers](/docs/migrating-from-docker/importing-saved-containers) into Podman Desktop using CLI and then access the [Docker Compatibility settings](/docs/migrating-from-docker/managing-docker-compatibility) to:
+
+- Check the system socket mapping status
+- On macOS: Use all Docker tools, including CLI, with the Podman engine
```

Comment:
Why is it restricted to MacOS ?

Reply:
Right now, this capability is not available on Windows. See this UX mockup screenshot: https://github.com/containers/podman-desktop/issues/8338#issuecomment-2367822941

The Third-Party Docker Tool Compatibility setting is only available on macOS, not on Windows.